vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -37,28 +37,11 @@ void call(Map parameters = [:]) {
     }
 }
 
-private boolean isYaml(String fileName) {
-    return fileName.endsWith(".yml") || fileName.endsWith(".yaml")
-}
-
-private boolean isProperties(String fileName) {
-    return fileName.endsWith(".properties")
-}
-
 private loadConfigurationFromFile(script, String configFile) {
 
-    String defaultPropertiesConfigFile = '.pipeline/config.properties'
     String defaultYmlConfigFile = '.pipeline/config.yml'
 
-    if (configFile?.trim()?.length() > 0 && isProperties(configFile)) {
-        Map configMap = readProperties(file: configFile)
-        script.commonPipelineEnvironment.setConfigProperties(configMap)
-    } else if (fileExists(defaultPropertiesConfigFile)) {
-        Map configMap = readProperties(file: defaultPropertiesConfigFile)
-        script.commonPipelineEnvironment.setConfigProperties(configMap)
-    }
-
-    if (configFile?.trim()?.length() > 0 && isYaml(configFile)) {
+    if (configFile) {
```

Comment:
Is it ok to now skip the check for the file extension?

Reply:
IMO the check for the file extension is not that cool anyhow. This does not assert that the file in fact contains valid yml. The other way: A file named e.g. xml can still contain valid yml. Strange, but possible. If somebody would like to choose another file extension, why not? Not a cool decission IMO, but well ...